homeassistant/components/media_player/services.yaml

```diff
@@ -181,6 +181,35 @@ browse_media:
       selector:
         text:
 
+search_media:
+  target:
+    entity:
+      domain: media_player
+      supported_features:
+        - media_player.MediaPlayerEntityFeature.SEARCH_MEDIA
+  fields:
+    media_content_type:
+      required: false
+      example: "music"
+      selector:
+        text:
+    media_content_id:
+      required: false
+      example: "A:ALBUMARTIST/Beatles"
+      selector:
+        text:
+    search_query:
```

Comment:
Does this automatically sort or should this be the first field?

Reply:
Agree should be first